src/components/Calendar/Calendar.tsx

```diff
@@ -28,6 +29,12 @@ const Calendar: React.FC<IProps> = ({ items }) => {
 
     return (
         <div className={styles.container}>
+            {offset > 0 &&
+                Array(offset)
+                    .fill(undefined)
+                    .map((_, i) => {
+                        return <Day key={i} mock={true} />
```

Comment:
https://react.dev/learn/rendering-lists

You might be tempted to use an item’s index in the array as its key. In fact, that’s what React will use if you don’t specify a key at all. But the order in which you render items will change over time if an item is inserted, deleted, or if the array gets reordered. Index as a key often leads to subtle and confusing bugs.